[2024-05-28T21:16:45.455+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T21:16:45.489+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_dag_1.python_task_2 manual__2024-05-28T21:16:40.570786+00:00 [queued]>
[2024-05-28T21:16:45.500+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_dag_1.python_task_2 manual__2024-05-28T21:16:40.570786+00:00 [queued]>
[2024-05-28T21:16:45.501+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-05-28T21:16:45.521+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): python_task_2> on 2024-05-28 21:16:40.570786+00:00
[2024-05-28T21:16:45.530+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=215) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-28T21:16:45.533+0000] {standard_task_runner.py:63} INFO - Started process 217 to run task
[2024-05-28T21:16:45.533+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_dag_1', 'python_task_2', 'manual__2024-05-28T21:16:40.570786+00:00', '--job-id', '2515', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp331lwicd']
[2024-05-28T21:16:45.537+0000] {standard_task_runner.py:91} INFO - Job 2515: Subtask python_task_2
[2024-05-28T21:16:45.600+0000] {task_command.py:426} INFO - Running <TaskInstance: example_dag_1.python_task_2 manual__2024-05-28T21:16:40.570786+00:00 [running]> on host 6ecac41592ab
[2024-05-28T21:16:45.733+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='gurlenko.13032001@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_dag_1' AIRFLOW_CTX_TASK_ID='python_task_2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T21:16:40.570786+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-28T21:16:40.570786+00:00'
[2024-05-28T21:16:45.736+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T21:16:45.750+0000] {base.py:84} INFO - Using connection ID 'coin_conn' for task execution.
[2024-05-28T21:16:48.467+0000] {logging_mixin.py:188} INFO - Current rate 68322.3165
[2024-05-28T21:16:48.483+0000] {python.py:237} INFO - Done. Returned value was: 68322.3165
[2024-05-28T21:16:48.485+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T21:16:48.508+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_dag_1, task_id=python_task_2, run_id=manual__2024-05-28T21:16:40.570786+00:00, execution_date=20240528T211640, start_date=20240528T211645, end_date=20240528T211648
[2024-05-28T21:16:48.544+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-28T21:16:48.567+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-28T21:16:48.571+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
